leaflet-image.js

```diff
@@ -30,6 +30,7 @@ module.exports = function leafletImage(map, callback) {
     // layers are drawn in the same order as they are composed in the DOM:
     // tiles, paths, and then markers
     map.eachLayer(drawTileLayer);
+    map.eachLayer(drawEsriDynamicLayer);
```

Comment:
This code should be included in index.js, not leaflet-image.js: leaflet-image.js is a generated file.